.gitignore

```diff
@@ -53,4 +53,12 @@ coverage.xml
 docs/_build/
 
 # Emacs backups
-*~
\ No newline at end of file
+*~
+
+# PyCharm config
+.idea/
+
+# Paths generated in testing
+django-pyfs/
+example/db.sql
+example/sample/static/
```

Comment:
It would be nice if these paths somehow indicated they were part of the test build.